validator/fastq.py

```diff
@@ -65,8 +69,7 @@ def _validate_plus(self, line):
 
     def _validate_quality_scores(self, line):
         for symbol in line:
-            if symbol not in (ord(value) for value in "!\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ \
-                [\]^_`abcdefghijklmnopqrstuvwxyz{|}~"):
+            if symbol not in self.quality_score_symbols:
```

Comment:
Here we can change the condition to if not(33 <= symbol <= 126), assuming the symbol is an ASCII byte value.